.github/workflows/cd.yml

```
@@ -14,7 +14,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v1
         with:
-          node-version: '14.x'
+          node-version: "18.x"
```

Lately, I prefer this style for no reason. 
```suggestion
          node-version: 18
```

Can you also upgrade `jsdom` (I assume node 14 was restricting.)
I will also change the Vercel deploy config later.

---

Got it 👍😆